xml/System/ArgumentNullException.xml

```diff
@@ -191,7 +191,7 @@
 |Property|Value|  
 |--------------|-----------|  
 |<xref:System.Exception.InnerException%2A>|A null reference (`Nothing` in Visual Basic).|  
-|<xref:System.ArgumentException.Message%2A>|A localized error message string that identifies the null argument. For example, if the `paramName` argument is "arg1", the English language message string is:<br /><br /> `Value cannot be null.Parameter name: arg01`|  
+|<xref:System.ArgumentException.Message%2A>|A localized error message string that identifies the null argument. For example, if the `paramName` argument is "arg1", the English language message string is (using .NET):<br /><br /> `Value cannot be null. (Parameter name: 'arg1')`|  
```

Comment:
```suggestion
|<xref:System.ArgumentException.Message%2A>|A localized error message string that identifies the null argument. For example, if the `paramName` argument is "arg1", the English language message string is:<br /><br />On .NET 5+ and .NET Core: `Value cannot be null. (Parameter name: 'arg1')`<br /><br />On .NET Framework: `Value cannot be null.\r\nParameter name: arg1`|  
```

Reply:
It's probably best to include the default message for both .NET Framework and .NET Core/5+.

Reply:
Yes having both is ideal. I couldn't figure out how one can build the docs site to preview the changes (the PR build preview is only accessible to MS I take it) so I didn't want to make more changes than necessary.

Reply:
Yes, the preview is internal only, unfortunately. I'll check it out and make sure it looks okay.